mmv1/third_party/terraform/tpgresource/common_diff_suppress.go.erb

```diff
@@ -179,25 +179,47 @@ func TimestampDiffSuppress(format string) schema.SchemaDiffSuppressFunc {
 	}
 }
 
-// suppress diff when saved is Ipv4 and Ipv6 format while new is required a reference
-// this happens for an internal ip for Private Services Connect
+// Suppresses diff for IPv4 and IPv6 different formats.
+// It also suppresses diffs if an IP and one reference are given.
```

Comment:
Clarifies that the diff is only suppressed if an IP is changed to a reference - otherwise it could be understood that a reference changing to an IP would be suppressed.

```suggestion
// It also suppresses diffs if an IP is changing to a reference.
```